src/code/credits.asm

```diff
@@ -2965,7 +2973,7 @@ CreditsLoadBGMap::
     add  hl, de                                   ; $5B23: $19
     ld   a, [hl+]                                 ; $5B24: $2A
     ld   b, a                                     ; $5B25: $47
-    ; Return bank to restore
+    ; return bank to restore
```

Comment:
This makes me think you've done a search/replace, and inadvertently changed the labels too